apps/examples/testcase/le_tc/kernel/kernel_tc_main.c

```diff
@@ -100,7 +100,7 @@ int kernel_tc_main(int argc, char *argv[])
 
 #ifdef CONFIG_TC_KERNEL_LIBC_PTHREAD
 #if (!defined CONFIG_PTHREAD_MUTEX_TYPES)
-#error CONFIG_PTHREAD_MUTEX_TYPES is needed for testing LIBC_PTHREAD TC
+	/* #error CONFIG_PTHREAD_MUTEX_TYPES is needed for testing LIBC_PTHREAD TC */
```

Comment:
Since `CONFIG_PTHREAD_MUTEX_TYPES is needed for testing LIBC_PTHREAD TC` is no more valid.
`CONFIG_PTHREAD_MUTEX_TYPES` is no more necessary.
If `libc_pthread_main` depends on `CONFIG_PTHREAD_MUTEX_TYPES`, we can use like previous.
But if not, we moved it to API tc level like this.
```
#ifndef CONFIG_CANCELLATION_POINTS
	tc_libc_pthread_pthread_setcanceltype();
	tc_libc_pthread_pthread_testcancel();
#endif
```